### PR TITLE
RC_Channel: rename EKF_POS_SOURCE to EKF_SOURCE_SET

### PR DESCRIPTION
**Pull request by peterbarker**
Co-authored-by: Michelle Rossouw <michelleros128@gmail.com>

these data sources span more than just position

### CODE REVIEW

**Comment by rmackay9** [comment]
Hi, this is fine with me.  Do we also want to change the RC_Channel parameter description?

// @Values{Copter, Rover, Plane, Blimp}: 90:EKF Pos Source

Of course if you change this the wiki will also need updating I think

**Comment by peterbarker** [comment]
```
Board                    AP_Periph  blimp  bootloader  copter  heli  iofirmware  plane  rover  sub
CubeOrange-periph-heavy  *                                                                     
Durandal                            *      *           *       *                 *      *      *
Hitec-Airspeed           *                                                                     
KakuteH7-bdshot                     *      *           *       *                 *      *      *
MatekF405                           *      *           *       *                 *      *      *
Pixhawk1-1M-bdshot                  *                  *       *                 *      *      *
f103-QiotekPeriph        *                                                                     
f303-Universal           *                                                                     
iomcu                                                                *                         
revo-mini                           *      *           *       *                 *      *      *
skyviper-v2450                                         *                                       
```

**Comment by peterbarker** [comment]
> Hi, this is fine with me. Do we also want to change the RC_Channel parameter description?

Done.

Also updated some comments in some scripting examples and applet.

> Of course if you change this the wiki will also need updating I think

Added WikiNeeded :-)

File: libraries/RC_Channel/RC_Channel.h [hunk 1]
```diff
@@ -191,7 +191,7 @@ class RC_Channel {
         CROW_SELECT =         87, // select CROW mode for diff spoilers;high disables,mid forces progressive
         SOARING =             88, // three-position switch to set soaring mode
         LANDING_FLARE =       89, // force flare, throttle forced idle, pitch to LAND_PITCH_DEG, tilts up
-        EKF_POS_SOURCE =      90, // change EKF position source between primary, secondary and tertiary sources
+        EKF_SOURCE_SET =      90, // change EKF data source set between primary, secondary and tertiary
```
Comment (rmackay9) [inline libraries/RC_Channel/RC_Channel.h:194]: thanks.  not that it matters much but I think of it as the sensor source set.

**Review by rmackay9 — APPROVED** [review]

**Pull request merged**